tarfs/mknod_stub.go

```diff
@@ -0,0 +1,28 @@
+// +build freebsd
```

Comment:
Ah, for some reason I thought you were going to make it a no-op. :man_facepalming: In that case let's call this file `mknod_freebsd.go` instead.

Reply:
No worries, I've renamed the file.